velox/functions/prestosql/Utf8Utils.cpp

```diff
@@ -100,6 +141,35 @@ int32_t tryGetCharLength(const char* input, int64_t size) {
     return -4;
   }
 
-  VELOX_UNREACHABLE();
+  if (size < 5) {
+    return -4;
+  }
+
+  auto filefthByte = input[4];
+  if (!utf_cont(filefthByte)) {
```

Comment:
qq, what does utf_cont return?

Reply:
UTF-8 continuation bytes starts with 0b10 (i.e., 0b10xxxxxx). utf_cont simply check whether the highest bits are 10.